### ISSUE TITLE
Assertion `groundtruth.size() == detections.size()' failed.

### ISSUE THREAD

**Issue by zhufeng888** [tomztyang/3DSSD#32]
I have trained a model and want to evaluate the trained weight. When I run lib/core/evaluator.py，I meet issues as follows:

**** Saving Evaluation results to the path /home/oiv/chenyanbin/3DSSD/lib/core/log/2020-11-25 00:36:59.354264 ****
**** Dataset length is 3384 ****
numpy applys seed 4209039388
Start data provider ipc://@dataflow-map-pipe-e02df190-b'0'
Starting Evaluation at 2020-11-25-17:37:24
2020-11-26 01:37:24.466919: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2020-11-26 01:37:24.686002: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1105] Found device 0 with properties: 
name: TITAN Xp major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:0a:00.0
totalMemory: 11.91GiB freeMemory: 11.76GiB
2020-11-26 01:37:24.686036: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1195] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: TITAN Xp, pci bus id: 0000:0a:00.0, compute capability: 6.1)
**** Evluation New Result ****
Assign From checkpoint: /home/oiv/chenyanbin/3DSSD/lib/core/log/2020-11-25 00:36:59.354264/model-79893
100%|██████████| 3384/3384 [06:13<00:00,  9.05it/s]
loadGroundtruth: no error!
loadDetections: no error!
**python /home/oiv/chenyanbin/3DSSD/lib/core/evaluator.py: evaluate.cpp:842: bool eval_class(FILE*, FILE*, CLASSES, const std::vector<std::vector<tGroundtruth> >&, const std::vector<std::vector<tDetection> >&, bool, bool, double (*)(tDetection, tGroundtruth, int32_t), std::vector<double>&, std::vector<double>&, std::vector<double>&, DIFFICULTY, METRIC): Assertion `groundtruth.size() == detections.size()' failed.**

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)

I have no idea about groundtruth.size() and detections.size(). Could you give me some advice?

**Comment by zhufeng888** [comment]
Ok， when I run on terminal instead of pycharm, it works. but the output result is hard to understand. Does there anyone konw the difinite meaning about the matrix shown as follows and I still want to konw how to work it on pycharm.

**** Evluation New Result ****
Assign From checkpoint: /home/hby/hdd/chenyanbin/3DSSD/lib/core/log/2020-11-25 00:36:59.354264/model-79893
100%|███████████████| 3384/3384 [05:45<00:00,  9.78it/s]
loadGroundtruth: no error!
loadDetections: no error!
precision_image:
[[ 0.96612436  0.93489265  0.92814064]
 [-1.         -1.         -1.        ]
 [-1.         -1.         -1.        ]]
precision_ground:
[[ 0.9327345   0.89477235  0.8677206 ]
 [-1.         -1.         -1.        ]
 [-1.         -1.         -1.        ]]
precision_3d:
[[ 0.91790676  0.83215886  0.80374014]
 [-1.         -1.         -1.        ]
 [-1.         -1.         -1.        ]]

**Comment by zhufeng888** [comment]
Oh~, the true reason is that I don't read label_2 correctly.
